rules/falco_rules.yaml

```diff
@@ -3194,6 +3194,16 @@
   priority: CRITICAL
   tags: [mitre_initial_access]
 
+- condition: open_read and (fd.name="/etc/passwd" and fd.nameraw contains "../.")
+  desc: etc passwd read via directory traversal
+  enabled: true
+  output: (user=%user.name user_loginuid=%user.loginuid program=%proc.name exe=%proc.exepath
+    command=%proc.cmdline parent=%proc.pname file=%fd.name fileraw=%fd.nameraw parent=%proc.pname
+    gparent=%proc.aname[2] container_id=%container.id image=%container.image.repository returncode=%evt.res)
+  priority: WARNING
+  rule: Directory traversal etc passwd read
```

Comment:
Hi, 
could you please follow the best practice on the rule syntax with the name on the top?
Thank you for the change and the contribution